src/views/catalog/CreateFromInstanceTypes/components/BootableVolumeList/BootableVolumeList.tsx

```diff
@@ -100,7 +112,7 @@ const BootableVolumeList: FC<BootableVolumeListProps> = ({
                 endIndex: prevPagination?.perPage,
               }));
             }}
-            loaded={loaded}
+            loaded={Boolean(loaded)}
             data={unfilteredData}
             // nameFilter={!displayShowAllButton && "modal-name"} can remove comment once this merged https://github.com/openshift/console/pull/12438 and build into new SDK version
```

Comment:
This PR was merged on March 13th. Have you checked to see if it has been built into a new SDK version?

Reply:
I checked we haven't thanks for reminding me to bother console folks :)